### DISCUSSION TITLE
招募远程职业前端一名，具体面议需要远程工作的留下联系方式

### DISCUSSION THREAD

**Discussion by stash-labs** [layui/layui#2073] [讨论]
### 议题条件

- [X] 我确认已查看官方使用文档：https://layui.dev ，但没有找到相关解决方案。
- [X] 我确认已在 [Issues](https://github.com/layui/layui/issues?q=is%3Aissue ) 中搜索过类似的问题，但没有找到相关解决方案。
- [X] 我已仔细阅读： 🍀[ Layui Issue 贡献指南](https://github.com/layui/layui/blob/main/CONTRIBUTING.md)

### 议题类型

功能请求

### 使用版本

v2.8.18

### 问题描述

招募远程职业前端一名，具体面议需要远程工作的留下联系方式

### 业务代码

```auto
招募远程职业前端一名，具体面议需要远程工作的留下联系方式
```

### 截图补充

前端招募

### 浏览器

Chrome

### 演示地址

123

### 友好承诺

- [X] 我承诺将本着相互尊重、理解和友善的态度进行交流，共同维护 Layui 良好的社区氛围。

**Comment by paogege0591** [comment]
用layui的应该大部分都是后端吧，专业前端谁还用jq啊。。。只需要干layui或者vue的活可以加我v：paowang250
